mantidproject/mantid
# IN4 instrument definition file update

This PR updates the IN4 instrument definition file (IDF) to more accurately describe the instrument. The file is actually generated by a script in the mantidgeometry repository; a [PR](https://github.com/mantidproject/mantidgeometry/pull/80) for that is also open, and the reviewer is requested to review it as well (updated IN6 IDF is already in the master).

[Here](https://github.com/mantidproject/documents/blob/master/Project-Management/ILL/Instrument_Definitions/IN4/IN4_Geometry.md) is a document describing the instrument geometry in detail.

Fixes #17992 

**To test:**

The IDF has been validated by the corresponding instrument scientists at ILL. So it should be enough to check that the instrument loads ([LoadEmptyInstrument](http://docs.mantidproject.org/nightly/algorithms/LoadEmptyInstrument-v1.html) should do the trick), everything shows up in the instrument view and that the IDF and the generating script look plausible.

**Release notes were updated accordingly.**

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
